### DISCUSSION TITLE
Livewire 3 Installation Command Fom Quickstart page is installing the previous version

### DISCUSSION THREAD

**Discussion by balduinofernando** [livewire/livewire#5806] [4. General]
Hi folks!

I was following the Guide from the quickstart page and the `composer require livewire/livewire` is installing the version 2.x.

![image](https://github.com/livewire/livewire/assets/20538071/e1c01b4d-8497-413e-960d-8e6f96febe5c)

![image](https://github.com/livewire/livewire/assets/20538071/4aec0944-8508-4dec-ba13-efc9ac2d3d87)

I guess it should be updated to match the new version.

**Comment by PhiloNL** [comment]
It's correctly mentioned in the installation docs:
https://livewire.laravel.com/docs/installation

Made a PR to update the quickstart (https://github.com/livewire/livewire/pull/5809)

**Comment by joshhanley** [comment]
This has been fixed in #5807. Thanks @balduinofernando 🙂
